Formula/wine.rb

```diff
@@ -38,6 +38,9 @@ def require_universal_deps?
     MacOS.prefer_64_bit?
   end
 
+  option "with-win64",
+         "Build with 64-bit support (only works with Xcode 8 and newer.)"
```

Comment:
Actually, it will work with 32-bit binaries. `wine` and `wine64` will run the appropriate program required for the executable (if 16-bit or 32-bit, `wine` is run, otherwise, if it is 64-bit, `wine64` is run.)